src/webui/webapplication.cpp

```diff
@@ -251,36 +247,36 @@ void WebApplication::doProcessRequest()
 
     const auto findAPICall = [&]() -> bool
     {
-        QRegularExpressionMatch match = m_apiPathPattern.match(request().path);
-        if (!match.hasMatch()) return false;
+            QRegularExpressionMatch match = m_apiPathPattern.match(request().path);
```

Comment:
I think you messed up the indention here and below.

Reply:
Outdated QtCreator broke it.